vane-trifles/src/main/java/org/oddlama/vane/trifles/FastWalkingListener.java

```diff
@@ -45,6 +50,8 @@ public void on_player_move(final PlayerMoveEvent event) {
 	public void on_entity_move(final EntityMoveEvent event) {
 		final var entity = event.getEntity();
 
+		if(entity.getType()  == EntityType.VILLAGER && villager_speedwalk == false) return;
```

Comment:
This is an issue I had when I was playing with Vane as well. In my opinion, it'd be better to allow restricting the speed to only players. It was even more annoying having creepers running at me or inescapable ravagers destroying me during a raid.

That, or *restrict* hostile mobs from gaining speed.

Reply:
I think the two most requested additional options were players-only and non-hostile, adding both shouldn't be a problem. I think think we should set the default to non-hostile because by far the most commonly reported issue was with hostile mobs being too strong. We can have a third exempt_villagers option if necessary, or a blocklist but that would be more effort to implement.